apps/condo/domains/billing/utils/validation.utils.js

```diff
@@ -14,9 +14,10 @@ const PAYMENT_SCHEMA = {
         recalculation: { type: 'string' },
         privilege: { type: 'string' },
         penalty: { type: 'string' },
+        volume: { type: 'string' },
     },
     required: ['formula'],
-    additionalProperties: false,
+    additionalProperties: true,
```

Comment:
Because we don't really know how are we going to change the schema

To know we need to ship billing feature to customers and then live with it for couple of weeks 

This way it's much cheaper to ship valid data from integrations. You simply need to add addiitonal property

Reply:
The frontend uses only fields depicted here, so there wont be a situation when frontend fails with error `something is undefined`